### ISSUE TITLE
Guidelines are not guidelines

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag3#493]
I know this is pedantic, but WCAG 3.0's guidelines aren't written as guidelines. A guideline is something to follow or do. But the "guidelines" in WCAG 3.0's latest draft are phrased as requirements or as expectations.

> "Users have equivalent alternatives for images."

Should be something like: 

> "Ensure users have equivalent alternatives for images."
